spec/feature_spec.rb

```diff
@@ -0,0 +1,33 @@
+require "my_inject"
+describe "Features" do 
+  describe "#my_inject" do
+    it "works with adding numbers together" do
+      a = [1,2,3,4]
+      expect(a.my_inject { |total, x| total + x } ).to eq 10
+    end
+    it "works with adding numbers together with an argument" do
+      a = [1,2,3,4]
+      expect(a.my_inject(2) { |total, x| total + x } ).to eq 12
+    end
+    it "works with other integer methods" do
+      a = [1,2,3,4,5,6,7,8,9,10]
+      expect( a.my_inject(1) { |total, x| x.even? ? total*x : total } ).to eq 3840
```

Comment:
Line is too long. [82/80]<br>Surrounding space missing for operator '*'.<br>Space inside parentheses detected.